02week/pigLatin.js

```diff
@@ -8,9 +8,56 @@ const rl = readline.createInterface({
 });
 
 
-function pigLatin(word) {
+function pigLatin(input) {
+  let inputStr = input.split('');
+  let counter  = 0;
 
-  // Your code here
+  inputStr.forEach(isVowel);
+  function isVowel(item,index){
+    if(index == 0)
+      console.log('Input letters are:');
+    console.log(item);
+  }
+  let index;
+  let newStr1;
+  let newStr2;
+  let vowelFound = false;
+  for(index = 0; index<inputStr.length;index++){
+    switch(inputStr[index]){
+      case 'a':
+      case 'e':
+      case 'i':
+      case 'o':
+      case 'u':
+      case 'A':
+      case 'E':
+      case 'I':
+      case 'O':
+      case 'U':
+           newStr1 = inputStr.slice(index);
+           newStr2 = inputStr.slice(0,index);
+           newStr1=newStr1.join('');
+           newStr2=newStr2.join('');
+           vowelFound = true;
+       break;
+    }
+    if(vowelFound)
```

Comment:
why are you looking if a vowel is found here and on line 48?

Reply:
Pushed new changes with refactored code

Reply:
https://github.com/bireswarb/javascript-workbook/pull/7